app/modules/intelligence/provider/provider_service.py

```
@@ -945,48 +1021,60 @@ async def call_llm_with_structured_output(
+                        "OLLAMA_API_KEY", "ollama"
+                    )
+                    client = instructor.from_openai(
+                        AsyncOpenAI(base_url=ollama_base_url, api_key=ollama_api_key),
+                        mode=instructor.Mode.JSON,
+                    )
+                    ollama_request_kwargs = {
+                        key: value
+                        for key, value in request_kwargs.items()
+                        if key not in {"base_url", "api_key", "api_version"}
+                    }
+                    response = await client.chat.completions.create(
```

Ruff (0.15.2)

[error] Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)